Rui_Xu/w2_d5/quiz

```diff
@@ -0,0 +1,13 @@
+1. document.getElementsByTagName('h1')[0];
+
+2. document.getElementsByTagName('h1')[0].textContent = 'Friend';
+
+3. document.getElementsByTagName('button')[0].addEventListener('click',function(){document.getElementsByTagName('div')[0].style.backgroundColor='red';});
+
+4.var favouriteColor = document.getElementById('favCol').value;
+
+5.document.getElementsByClassName('my-articles')[0].innterHTML="<p></p>";
```

Comment:
Actually, upon further investigation (and to my surprise!) this is actually correct and **will** correctly create a DOM node. That having been said, append is still the recommended method, because you **cannot** append additional nodes using innerHTML.

However, this satisfies the conditions of the question. Let the records state, this one is correct!